tree-sitter-stack-graphs/src/cli/analyze.rs

```diff
@@ -59,6 +59,10 @@ pub struct AnalyzeArgs {
     #[clap(long, short = 'v')]
     pub verbose: bool,
 
+    /// Hide failure error details.
+    #[clap(long)]
+    pub hide_error_details: bool,
```

Comment:
I feel like booleans should be phrased in terms of truth rather than falsity, so I would prefer `show_error_details` and have it set to true on line 85 below.

Reply:
I would agree if this was only a field, but it is also a command line flag. I think those should necessarily be modifications on the default behavior (which is to show, in this case)?

Reply:
Indeed, yeah. I agree that the default behavior shouldn’t be changed. I was envisioning `./tree-sitter-stack-graphs --show-errors=false`? Not a huge deal.

Reply:
I see. I'm not too fond of that, tbh (unless not providing a value is different from providing either true or false).

Reply:
Absolutely.